pkg/controller/init.go

```diff
@@ -636,6 +636,30 @@ func (c *Controller) initSyncCrdSubnets() error {
 	return nil
 }
 
+func (c *Controller) initSyncCrdVpcNatGw() error {
+	klog.Info("start to sync crd vpc nat gw")
+	cm, err := c.configMapsLister.ConfigMaps(c.config.PodNamespace).Get(util.VpcNatGatewayConfig)
+	if err != nil && !k8serrors.IsNotFound(err) {
+		klog.Errorf("failed to get ovn-vpc-nat-gw-config, %v", err)
+		return nil
```

Comment:
should return err here, as something wrong with apiserver

Reply:
fixed